src/Pages/Repositories/ContentListTable/components/ContentListFilters.tsx

```diff
@@ -64,7 +64,7 @@ const useStyles = createUseStyles({
   },
 });
 
-const statusValues = ['Invalid', 'Pending', 'Unavailable', 'Valid'];
+const statusValues = ['Invalid', 'In progress', 'Unavailable', 'Valid'];
```

Comment:
since the API only recognizes `pending` and not `in progress` as a status, we'd also need to map that here, something like this:

```
const statusToApiMap: Record<string, string> = {
  'In progress': 'Pending',
};
```

then in the useEffect that calls setFilterData, you can map to the correct status in the api filters:

```
useEffect(() => {
    setFilterData({
      searchQuery: debouncedSearchQuery,
      versions: getLabels('version', debouncedSelectedVersions),
      arches: getLabels('arch', debouncedSelectedArches),
      statuses: debouncedSelectedStatuses.map((status) => statusToApiMap[status] || status),
    });
  }, [
    debouncedSearchQuery,
    debouncedSelectedVersions,
    debouncedSelectedArches,
    debouncedSelectedStatuses,
  ]);
```